### ISSUE TITLE
Local elections and party affiliation

### ISSUE THREAD

**Issue by evz** [mysociety/yournextrepresentative#512]
It seems that YNR has a pretty strong assumption that each candidate running for an office is affiliated with a particular political party. This is often not the case with local elections in the US (in Chicago for instance, our city council is technically non-partisan). In the instance we are setting up for St. Paul, there are candidates who seem to espouse a particular party's ideology but are not supported by or officially associated with the larger organization. 

I'm not sure how much this breaks things in a general sense but I thought I'd bring it up as I'm pretty sure this is something that is not unique to these two places.

**Comment by mhl** [comment]
In the UK we had to create some "pseudo-parties", like "Independent", and in Argentina we also had to create an "Unknown" pseudo-party.

**Comment by mhl** [comment]
This isn't an ideal solution, but it's probably the best workaround without significant changes to how we handle parties. Another ticket that relates to the party / affiliation description not being flexible enough is: https://github.com/mysociety/yournextrepresentative/issues/456 - ideally we should display the hierarchies of party-like organizations on the site.

**Comment by evz** [comment]
Another way of addressing it might be to have an election flagged as "non-partisan" and that cascades down to the way that the candidates are stored. Not sure how that would work in a technical sense but it would make it easier to sidestep this issue for a whole election instead of having kind of an awkward set of instructions given to people filling out the info to "ignore" the party field or to set it to some arbitrary value.

**Comment by tfgg** [comment]
Indeed, my reading of this issue is that in some elections _no-one_ belongs to a party, so that part of the UI is redundant and can be turned off, which sounds fairly straight forward.

**Comment by mhl** [comment]
I like this idea, for the record - having a "non-partisan" flag on an election that turns off the party-related UI.
